scripts/update-xliff.py

```diff
@@ -50,15 +50,23 @@ def main():
 
     parser = argparse.ArgumentParser()
     parser.add_argument('base_folder', help='Path to folder including subfolders for all locales')
+    parser.add_argument('locales', default='*', nargs='*', help='Locales to process (default: %(default)s)')
```

Comment:
@st3fan yeah my fault.  Probably would have been better to check if `locales` is set then use its values, otherwise iterate over all langs.